cmd/scheduler/cmd/root.go

```diff
@@ -41,7 +41,7 @@ var (
 var rootCmd = &cobra.Command{
 	Use:   "scheduler",
 	Short: "the scheduler of dragonfly",
-	Long: `Scheduler is a long-running process which receives and manages download tasks from the dfdaemon, notify the seed peer to return to the source, 
+	Long: `Scheduler is a long-running process which receives and manages download tasks from the dfdaemon, notify the seed peer to return to the source,
```

Comment:
This is because my editor is triming trailing whitespace automatically, I think this change is ok, since there is no difference for human when reading the help output(it wraps line here).